e2e/api/api_test.go

```diff
@@ -186,3 +188,144 @@ func (s *APITestSuite) TestLogoutRemovesCells() {
 	_, err = s.api.JailManager().Cell(testChatID)
 	require.Error(err, "Expected that cells was removed")
 }
+
+func (s *APITestSuite) TestCreateChildAccount() bool {
```

Comment:
Please compare this with `e2e/accounts/accounts_test.go` as a lot of test cases repeats here.

Reply:
Tests are basically the same.  I think the e2e test probably belong on the StatusAPI that way if any additional logic is added to StatusAPI the test will still cover it.  

Should I delete the duplicated test in e2e/accounts as part of this PR? Or remove the new test from api?